src/issuemd-core.js

```diff
@@ -2,42 +2,51 @@
 
 ! function () {
```

Comment:
how about...

     ! function (root) {

... and at bottom of file...

    }(this);